src/main/java/org/team4909/powerup2018/Robot.java

```diff
@@ -94,17 +97,20 @@ protected void subsystemInit() {
         elevator = new ElevatorSubsystem(
                 new BionicSRX(
                         3, false,
-                        FeedbackDevice.CTRE_MagEncoder_Relative, true,
+                        FeedbackDevice.CTRE_MagEncoder_Relative, false,
                         0.28, 0, 0,
                         3
                 ),
-                manipulatorGamepad, BionicF310.LY, -.6,
+                manipulatorGamepad, BionicF310.LY, -0.45,
                 33150
         );
+
+        /**
```

Comment:
The special comment start prefix `/**` (with two asterisks) is an indication to the JavaDoc documentation system that the contents of the comment are to be placed into the documentation, and the comment is expected to be in a certain format. Using this comment start prefix for non-JavaDoc comments may lead to confusion -- both by the compiler and by people reading the code and wondering what it's supposed to be documenting. It is, therefore, not generally a good idea to begin normal comments (or commented-out code) with this prefix.

My policy in this regard is based on how long I expect it to be commented out rather than removed. If it's temporarily commented and I intend to soon delete it, I'll just use a normal comment with `/*` and `*/`. On the other hand, if it's valuable code to retain, as it demonstrates how to do something, I'll enclose it in `if (false) {` and `}`. In the latter case, the code inside of the `if (false)` block should be well commented to indicate what it is demonstrating, and thus, why the code is retained. This latter method can also be used if you remove some code that caused a problem, and you want to say to future readers, "Hey, don't do this!" But then, too, it should be well-commented as to what the problem was.